src/IssueViz.Security.UnitTests/Commands/HotspotsToolWindowCommandTests.cs

```diff
@@ -82,90 +75,50 @@ public void Ctor_CommandAddedToMenu()
         }
 
         [TestMethod]
-        public void Execute_WindowNotCreated_NoException()
+        public void Execute_ServiceCalled()
```

Comment:
This and the following tests are completely new tests; I suggest ignoring the old tests completely and review them as new code.